File: June-18/12-06-18/java/01-Easy-multiple.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        while(t-- > 0)
        {
            int n = input.nextInt();
            n  -= 1;
            long x = n / 3;
            long y = n / 5;
            long z = n / 15;
            
            long s1 = 3 * (x * (x+1))/ 2;
            long s2 = 5 * (y * (y+1))/ 2;
            long s3 = 15 * (z * (z+1))/ 2;
            long ans = s1 + s2 - s3;
            System.out.println(ans);
        }
        
    }
}

File: June-18/12-06-18/python/01-Easy-multiple.py
for _ in range(int(input())):
    n = int(input())
    n -= 1
    x = n // 3
    y = n // 5
    z = n // 15
    
    s1 = 3 * (x * (x+1)) // 2
    s2 = 5 * (y * (y+1)) // 2
    s3 = 15 * (z * (z+1)) // 2
    ans = s1 + s2 - s3
    print(ans)
